vnext/Desktop/Modules/WebSocketModule.cpp

```diff
@@ -13,12 +13,13 @@
 using namespace facebook::xplat;
 using namespace folly;
 
+using Microsoft::Common::Unicode::Utf8ToUtf16;
+
```

Comment:
@JunielKatarn Please remove all the clang-format-off, I don't feel comfortable signing off on turning off clang even for blocks of code. We have a set of clang rules we are all following. If anybody has a case to make about changing the rules, I'm all for that, but I don't want us to special case any particular piece of code because of personal preference, as this is a team product.